pkg/unikontainers/unikernels/unikernel.go

```diff
@@ -33,29 +36,21 @@ type UnikernelParams struct {
 
 var ErrNotSupportedUnikernel = errors.New("unikernel is not supported")
 
-func UnikernelCommand(unikernelType UnikernelType, data UnikernelParams) (string, error) {
+func New(unikernelType UnikernelType) (Unikernel, error) {
 	switch unikernelType {
 	case RumprunUnikernel:
-		unikernel, err := newRumprun(data)
+		unikernel, err := newRumprun()
 		if err != nil {
-			return "", err
+			return nil, err
 		}
-		command, err := unikernel.CommandString()
-		if err != nil {
-			return "", err
-		}
-		return command, nil
+		return unikernel, nil
 	case UnikraftUnikernel:
-		unikernel, err := newUnikraft(data)
-		if err != nil {
-			return "", err
-		}
-		command, err := unikernel.CommandString()
+		unikernel, err := newUnikraft()
```

Comment:
Likewise, the error returned by newUnikraft() will always be nil, so it could be omitted.

Reply:
I will fix it.